### PR TITLE
sysding: add in setup_ipmp creation of ipmp group the old fashioned way

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

**Comment by olbohlen** [comment]
looks good to me

**Comment by olbohlen** [comment]
oh sorry. please fix also then the BUGS section of the man-page

**Comment by grueni** [comment]
I edited the paragraph for ipmp and added some text at the end. Please make a proof read.

File: sysding.1m [hunk 1]
```diff
@@ -259,5 +259,5 @@ reviewed after installation and used for other installations (like non-global zo
 .SH BUGS
 .sp
 .LP
-\fBsetup_ipmp\fR does not work on illumos based OS, since it depends on 
-\fBipadm create-ipmp\fR which is currently not implemented.
\ No newline at end of file
+Currently no bugs are known. 
+If you find a bug or have a feature request open an issue at https://github.com/OpenIndiana/sysding.
```
Comment (pyhalov) [inline sysding.1m]: Please, refer to https://bugs.openindiana.org as https://github.com/OpenIndiana/sysding doesn't support creating issues.

**Comment by pyhalov** [comment]
@grueni Sorry, I've missed this PR. Can you fix man page so that we could merge it?

**Comment by pyhalov** [comment]
@olbohlen Can you look at this? 
If I don't get negative feedback in 2 days, I'll merge it.

**Comment by olbohlen** [comment]
LGTM, ship it!

**Pull request merged**